README.md

```diff
@@ -1,21 +1,36 @@
 # ae-channel-service
 
-The [channel_runner](apps/ae_socket_connector/lib/channel_runner.ex) is to be considered as a `state-channel` client, which will execute several state channel operations, as token transfers and off-chain contract calls.
+The channel service is to be considered as a `state-channel` client implementation, which will execute several state channel operations, as token transfers and off-chain contract calls.
 
 This implementation benefits by being able calling erlang functions provided by the core team.
 
+Read up on state channels [here](https://github.com/aeternity/protocol/blob/master/node/api/channels_api_usage.md)
+
 ## Build
 
 ```
 make clean deps
 make shell
 ```
 
-## Configure
+## Configure - local node
+
+add valid accounts in [apps/ae_socket_connector/lib/test_accounts.ex](apps/ae_socket_connector/test/accounts_test.exs)
+
+These account must exist on the node.
+
+add the address to your [æternity](https://github.com/aeternity/aeternity) node and your network id in [apps/ae_socket_connector/lib/client_runner.ex](apps/ae_socket_connector/lib/client_runner.ex#L8)
```

Comment:
Hint: You should use [Elixir.Config](https://hexdocs.pm/elixir/Config.html) for this which you already have in place anyway.